pages/advanced-algorithms/available-algorithms/periodic.mdx

```diff
@@ -85,6 +85,50 @@ YIELD success, number_of_executed_batches
 RETURN success, number_of_executed_batches;
 ```
 
+### `delete()`
```

Comment:
iterate and delete are h3 and Procedures title is h3. Fix so it's the same as on other module pages.